pkgs/test_core/lib/src/runner/plugin/remote_platform_helpers.dart

```diff
@@ -30,10 +30,16 @@ import 'package:test_api/src/suite_channel_manager.dart'; // ignore: implementat
 ///
 /// If [beforeLoad] is passed, it's called before the tests have been declared
 /// for this worker.
-StreamChannel serializeSuite(Function Function() getMain,
+StreamChannel<Object> serializeSuite(Function Function() getMain,
         {bool hidePrints = true, Future Function()? beforeLoad}) =>
-    RemoteListener.start(getMain,
-        hidePrints: hidePrints, beforeLoad: beforeLoad);
+    RemoteListener.start(
+      getMain,
+      hidePrints: hidePrints,
+      beforeLoad: beforeLoad,
+    )
+    // TODO: this cast should not be needed
+    // and yet it fails at SDK dc1d7c87205 without it
+    as StreamChannel<Object>;
```

Comment:
This is a head scratcher, but absolutely necessary to get things working!

Reply:
What do you mean by "fails"? At runtime? Statically?

Reply:
```
  Failed to load "test/isolate_channel_test.dart":
  Unable to spawn isolate: ../test/pkgs/test_core/lib/src/runner/plugin/remote_platform_helpers.dart:35:20: Error: A value of type 'StreamChannel<dynamic>' can't be assigned to a variable of type 'StreamChannel<Object>'.
   - 'StreamChannel' is from 'package:stream_channel/stream_channel.dart' ('lib/stream_channel.dart').
   - 'Object' is from 'dart:core'.
      RemoteListener.start(getMain,
                     ^
```

Reply:
statically!

Reply:
Is one of these libraries not opted in, and the other is?

Reply:
I'm running w/ the bleeding SDK bits that won't let things be mixed – both are opted-in

Reply:
Turns out this **is not** needed. User error...